app/uk/gov/hmrc/taxenrolmentassignmentfrontend/views/EnrolCurrentUser.scala.html

```diff
@@ -0,0 +1,86 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.taxenrolmentassignmentfrontend.views.html.Layout
+@import uk.gov.hmrc.taxenrolmentassignmentfrontend.models.forms.EnrolCurrentUserId
+
+@this(
+    layout: Layout,
+    govukRadios : GovukRadios,
+    govukErrorSummary: GovukErrorSummary,
+    govukWarningText : GovukWarningText,
+    govukButton: GovukButton
+)
+
+@(form: Form[EnrolCurrentUserId], currentUserId: String, optSAUserId: Option[String] = None)(implicit request: Request[AnyRef], messages: Messages)
+
+@formError = @{form.errors.map(_.message).headOption}
+
+@layout(
+    pageTitle = Some(messages("enrolCurrentUser.title")), Some("?")) {
+
+    @if(formError.isDefined) {
+        @govukErrorSummary(ErrorSummary(
+          classes = "validation-summary govuk-!-display-none",
+          errorList = Seq(
+          ErrorLink(
+            href = Some("#use-current-id-error"),
+            content = Text(messages(formError.get))
+          )
+        ),
+          title = Text(messages("validation.summary.heading")),
+          attributes = Map("id" -> "errors")
+        ))
+    }
+
+    @govukRadios(Radios(
+        fieldset = Some(Fieldset(
+          legend = Some(Legend(
+          content = Text(messages("enrolCurrentUser.heading")),
+          classes = "govuk-fieldset__legend--l",
+          isPageHeading = true
+          ))
+        )),
+        errorMessage = formError.map(error => ErrorMessage(
+          content = Text(messages("ers_check_file_type.alert"))
```

Comment:
Assuming this error key is meant to be something else?

Reply:
yep